src/components/modules/ui.ts

```diff
@@ -356,6 +373,48 @@ export default class UI extends Module<UINodes> {
     }, {
       passive: true,
     });
+
+    /**
+     * Start watching 'block-hovered' events that is used by Toolbar for moving
+     */
+    this.watchBlockHoveredEvents();
+  }
+
+  /**
+   * Listen redactor mousemove to emit 'block-hovered' event
+   */
+  private watchBlockHoveredEvents(): void {
+    /**
+     * Used to not to emit the same block multiple times to the 'block-hovered' event on every mousemove
+     */
+    let blockHoveredEmitted;
+
+    this.readOnlyMutableListeners.on(this.nodes.redactor, 'mousemove', _.throttle((event: MouseEvent | TouchEvent) => {
```

Comment:
Why not `mouseenter` and `mousemove`. Also would be great to check on pointer devices and mobile devices with connected touchpad/mouse

Reply:
The `mouseenter` would have to be bound to every Block, and there are can be a large amount of them. I think it's better to use the single event on the `redactor` node.

> Also would be great to check on pointer devices and mobile devices with connected touchpad/mouse

I will test the RC version in [CodeX Docs](https://github.com/codex-team/codex.docs), so I will try to use various hardware.